aptos-move/framework/aptos-stdlib/sources/type_info.spec.move

```diff
@@ -1,8 +1,28 @@
 spec aptos_std::type_info {
     // Move Prover natively supports `type_of` and `type_name`.
 
-    spec chain_id_internal {
-        // TODO: temporary mockup.
+    spec fun spec_is_primitive_type<T>(): bool {
```

Comment:
Do not use this approach to test whether a `T` is not a struct, instead, defined a spec function
```
spec native fun spec_is_struct<T>(): bool;
```

See the test case for an example:
https://github.com/move-language/move/blob/main/language/move-prover/tests/sources/functional/type_reflection.move#L13

Reply:
Thanks! Updated using `spec_is_struct`.